app/pages/viewuser/confirmation/contextCreator.js

```diff
@@ -0,0 +1,10 @@
+import manifest from './manifest.json';
+import { baseUrl } from '../../../config';
+
+export const getContext = ({
+  user, organisationId = 1, status,
```

Comment:
the organisationId is default here to 1. Should that be there?

Reply:
nope!